loops.js

```diff
@@ -11,7 +11,13 @@
 // Dit heb je nog niet geleerd, maar bekijk hiervoor dit MDN-artikel eens: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat
 // ==========================================
 
-
+console.log("Opdracht 1:")
+console.log("-------------------------------")
+for (let i = 0; i < 6; i++) {
```

Comment:
je begint hier met een lege console.log, dit is op te lossen door "*".repeat(i + 1); te loggen, of de loop te beginnen met "let i = 1"

Reply:
Scherp, ik heb het echt compleet over het hoofd gezien dat hij een lege regel logde!